xml/operations-configuring-identity-identity_limitations.xml

```diff
@@ -90,6 +90,51 @@
     </para>
    </listitem>
   </itemizedlist>
+  <para>
+   <emphasis role="bold">Hash Algorithm Configuration</emphasis>
+  </para>
+  <itemizedlist>
+    <listitem>
+     <para>
+     The default hash algorithm is <literal>bcrypt</literal>, which has a built-in
+     limitation of 72 characters. As keystone defaults to a secret length of 86
+     characters, customers may choose to change the keystone hash algorithm to one
+     that supports the full length of their secret.
+    </para>
+   </listitem>
+   <listitem>
+     <para>Process for changing the hash algorithm configuration:</para>
+     <procedure>
+       <step>
+         <para>
+           Update the <literal>identity</literal> section of keystone.conf.j2 to reference the desired algorithm
+         </para>
+<screen>
+[identity]
+password_hash_algorithm=sha512_crypt
```

Comment:
We should use "pbkdf2_sha512" instead of "sha512_crypt" here as "sha512_crypt" is not in the supported config.
See https://opendev.org/openstack/keystone/src/branch/stable/rocky/keystone/conf/identity.py#L113

Reply:
good catch, I will fix that, rebuild and repush... I may also squash the cherry-pick into the base commit

Reply:
fix pushed